lib/Connection.js

```diff
@@ -29,6 +29,13 @@ function Connection(router, req, res) {
         if (options) {
             if (options.props) request.setProps(options.props);
             if (options.resource) request.setResource(options.resource);
+
+            if (options.query) {
+                for (var key in options.query) {
```

Comment:
I see.  I had thought it was the way the URL would be passed in, as opposed to construction of URL with the connection object we get.  My bad.